### PR TITLE
Remove timestamps in logs, gardener does it already. #3002

### PR DESCRIPTION
**Pull request by estellecomment**
medic/medic-webapp#3002

### CODE REVIEW

File: lib/logger.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var winston = require('winston');
 
 var logger = new (winston.Logger)({
     transports: [
-        new (winston.transports.Console)({'timestamp':true})
+        new (winston.transports.Console)()
```
Comment (garethbowen) [inline lib/logger.js]: I think this is the default configuration so you probably don't need to provide anything. Try replacing this with... `var logger = new (winston.Logger);` or something?

Comment (alxndrsn) [inline lib/logger.js]: Those brackets look weird to me.  Would `new winston.transports.Console()` be incorrect?

File: server.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ var config = require('./config'),
 
 if (arg === 'debug') {
     logger.info('setting loglevel to %s.', arg);
-    logger.transports.console.level = arg;
```
Comment (alxndrsn) [inline server.js]: Is this change related to date duplication?

**Comment by alxndrsn** [comment]
Looks fine, but:

* I don't understand how the `console`/`Console` typo is related
* I suspect the log output will still duplicate the level, e.g.

### Previous

```2016-12-08T17:19:35.878Z - info: 2016-12-08T17:19:35.877Z - info: startup complete.```

### Fixed

```2016-12-08T17:19:35.878Z - info: info: startup complete.```

^^ will `info` appear twice?

File: server.js (hunk 2 above)
Comment (estellecomment) [inline server.js]: Weirdly, yes. 
Using the default logger, in `lib/logger`, seems to create the `logger.transports` object differently, and `console` is undefined, whereas `Console` is there. 
If you create the logger with arguments (as was done up to now) you get `console`.

File: server.js [hunk 3]
```diff
@@ -20,8 +20,8 @@ config.init(function(err) {
     }
     logger.info('loaded config.');
     if (!arg) {
-        logger.transports.console.level = config.get('loglevel');
-        logger.debug('loglevel is %s.', logger.transports.console.level);
```
Comment (alxndrsn) [inline server.js]: ?  I didn't just make this comment...

Comment (alxndrsn) [inline server.js]: It was 5 days ago

**Comment by estellecomment** [comment]
@alxndrsn your comments about log levels make sense, I agree it would be better, but I'm not finding the time to get to this, so I'm just going to merge. Let me know if you actually really care. 

Is the Console thing clear?

**Comment by estellecomment** [comment]
yeah, one comes from gardener, the other from api/sentinel. 
Cool, thanks!
